editable-span/editable-span-test.js

```diff
@@ -18,10 +18,15 @@ describe("editable-span", () => {
 			el.edit(ev);
 
 			assert.ok(el.editing, "edit() sets editing to true");
+			el.connect(); // start listeners
+			el.render(); // ensure the span child is available
```

Comment:
`connect` will automatically call `render` internally. You could just call `connect` above instead of `initialize`.

Reply:
Good catch.  I wasn't aware of that and I'll pare this down to just `connect`